### PR TITLE
Dismiss old tooltip before triggering a new one

### PR DESCRIPTION
**Pull request by labkey-tchad**
#### Rationale
Test sometimes has two tooltips showing. Need to try not to trigger multiple tooltips.

#### Related Pull Requests
* N/A

#### Changes
* Dismiss old tooltip before triggering a new one
* Clean up some IntelliJ warnings

### CODE REVIEW

**Review by labkey-sweta — COMMENTED** [review]
Test is still failing

File: test/src/org/labkey/test/components/targetedms/UtilizationCalendarWebPart.java [hunk 1]
```diff
@@ -22,7 +22,7 @@ public UtilizationCalendarWebPart(WebDriver driver)
 
     public void waitForPage()
     {
-        getWrapper().waitFor(() -> elementCache().yearTitle.isDisplayed(this),
+        WebDriverWrapper.waitFor(() -> elementCache().yearTitle.isDisplayed(this),
                 "Calendar did not load on time",
```
Comment (labkey-sweta) [inline test/src/org/labkey/test/components/targetedms/UtilizationCalendarWebPart.java:26]: What is the difference between getWrapper wait and WebDriverWrapper wait?

File: test/src/org/labkey/test/tests/targetedms/TargetedMSUtilizationCalendarTest.java [hunk 2]
```diff
@@ -20,7 +20,7 @@ public class TargetedMSUtilizationCalendarTest extends TargetedMSTest
     @Override
     protected @Nullable String getProjectName()
     {
-        return getCurrentTestClass().getSimpleName() + " Project";
+        return TargetedMSUtilizationCalendarTest.class.getSimpleName() + " Project";
     }
```
Comment (labkey-sweta) [inline test/src/org/labkey/test/tests/targetedms/TargetedMSUtilizationCalendarTest.java:24]: Curious, how is it different?

File: test/src/org/labkey/test/components/targetedms/UtilizationCalendarWebPart.java (hunk 1 above)
Comment (labkey-tchad) [reply test/src/org/labkey/test/components/targetedms/UtilizationCalendarWebPart.java:26]: `WebDriverWrapper.waitFor` is a static method. Referencing it through the non-static `getWrapper` is allowed in Java but is misleading.

File: test/src/org/labkey/test/tests/targetedms/TargetedMSUtilizationCalendarTest.java (hunk 2 above)
Comment (labkey-tchad) [reply test/src/org/labkey/test/tests/targetedms/TargetedMSUtilizationCalendarTest.java:24]: In reality, it probably doesn't ever evaluate to something different but `getCurrentTestClass` is a static method in BaseWebDriverTest that could return `null`.
`getClass()` is actually the method that this should have used (I just changed it).

**Comment by labkey-tchad** [comment]
Unrelated tests are failing now

**Review by labkey-sweta — APPROVED** [review]

**Pull request merged**